### PR TITLE
[functions] Fix typos in exceptions related to functions

### PR DESCRIPTION
**Pull request by vzhikserg**
### Modifications

Fix typos in exceptions related to functions. The tests were updated as well.

### CODE REVIEW

File: pulsar-functions/utils/src/main/java/org/apache/pulsar/functions/utils/FunctionConfigUtils.java [hunk 1]
```diff
@@ -35,8 +35,6 @@
 import java.lang.reflect.Type;
 import java.net.MalformedURLException;
 import java.util.*;
```
Comment (jiazhai) [inline pulsar-functions/utils/src/main/java/org/apache/pulsar/functions/utils/FunctionConfigUtils.java:37]: @vzhikserg Thanks for the fix. Would you please also help remove this `*` use case? we should avoid `*`

Comment (vzhikserg) [reply pulsar-functions/utils/src/main/java/org/apache/pulsar/functions/utils/FunctionConfigUtils.java]: Sure, it is fixed

**Pull request merged**